src/main/java/com/microsoft/sqlserver/jdbc/SQLServerDataSource.java

```diff
@@ -225,6 +225,19 @@ public GSSCredential getGSSCredentials() {
                 SQLServerDriverObjectProperty.GSS_CREDENTIAL.getDefaultValue());
     }
 
+    @Override
+    public void setUsePlatformGSSCredentials(boolean usePlatformGSS) {
+        setObjectProperty(connectionProps, SQLServerDriverBooleanProperty.USE_PLATFORM_GSS_CREDENTIALS.toString(),
+                usePlatformGSS);
+    }
+
+    @Override
+    public boolean getUsePlatformGSSCredentials() {
+        return getBooleanProperty(connectionProps,
+                SQLServerDriverBooleanProperty.USE_PLATFORM_GSS_CREDENTIALS.toString(),
+                SQLServerDriverBooleanProperty.ENCRYPT.getDefaultValue());
```

Comment:
Change it to `SQLServerDriverBooleanProperty.USE_PLATFORM_GSS_CREDENTIALS.getDefaultValue()`.